lib/mix/tasks.ex

```diff
@@ -47,6 +48,15 @@ defmodule Mix.Tasks.Coveralls do
         all_options
       end
 
+    all_options =
```

Comment:
this seems a bit unnecessary just to massage the option to the name we want. we could just do `flag_name: options[:flagname]` in the other tasks, right?

Reply:
good catch!

Reply:
https://github.com/duffelhq/excoveralls/pull/1/commits/a80c5e257a21593d06750580833e172b707bb0b3

Fixed on this end :)